csharp/ApiDocs/.gitignore

```diff
@@ -7,4 +7,6 @@
 /**/bin/
 /**/obj/
 _site
-dest
\ No newline at end of file
+dest
+csharp
+*.txt
```

Comment:
Is this for something that gets generated? Can you reply in the comments for the record, what specifically. Thanks!

Reply:
Yes, these get generated when you build the project. When you build docfx it generates an out folder with the static site. The folder name is specified in the docfx.json file.